build/validate-bicep.sh

```diff
@@ -29,7 +30,7 @@ do
         exec 3>&-
     fi
     
-    if [[ ! $EXITCODE -eq 0 || ! -z $STDERR ]]
+    if [[ ! $EXITCODE -eq 0 || (! -z $STDERR && ! $STDERR == $WARNING_MSG* ) ]]
```

Comment:
did you test this? I don't speak bash-fu

Reply:
yea, ran this through a workflow_dispatch on the branch and it passes